### PR TITLE
Adds timeout for initial element.

### PR DESCRIPTION
**Pull request by zorkow**
That could take care of the initial message problem. It was indeed a timing issue. 
Tested with Win7/IE10/NVDA. 
@pkra could you please have a look at other combinations?

### CODE REVIEW

**Comment by pkra** [comment]
Will do.

**Comment by pkra** [comment]
I can confirm that NVDA now catches the first speech string on Win7 FF, Chrome. JAWS does not.

But more seriously, I cannot walk at all using that branch. Do I need to merge other branches into this?

**Comment by pkra** [comment]
FWIW, merging the other PRs locally did not help with the broken walker (though I get earcons so the walker must think it's at an edge)

**Comment by pkra** [comment]
I just realized that JAWS did not have the initial-speech-string problem in the past :-(

On Wind 10, JAWS only voices the initial speech string on Chrome.

**Comment by pkra** [comment]
Hm... I think the support matrix is wrong and JAWS did not voice the initial speech in FF on Win 10. At least I can't get it to with the current main branch.

**Comment by pkra** [comment]
Were there changes to SRE?

**Comment by zorkow** [comment]
Walker should work. It works for me at least.

**Comment by zorkow** [comment]
Apart from the bug fix for the embellished fences, there were no changes to SRE.

**Comment by pkra** [comment]
The walker does not work for me on any browser or OS anymore (on https://mathjax.github.io/MathJax-RespEq/Semantics-Lab/Semantics-Lab-TeX.html)

**Comment by pkra** [comment]
FWIW, with or without screenreaders it does not work.

The same applies to the `timeout_first_message` branch, with or without the other branches merged in.

**Comment by zorkow** [comment]
It works perfectly fine for me. On all branches. Sorry I am at a loss, as I am not sure where I could start the debugging.
@dpvc do you have any problems?

**Comment by zorkow** [comment]
Just tried Chrome and Safari on the missus's MacBook. Both worked fine.

**Comment by pkra** [comment]
D'oh! Of the highest order. I was stuck on the dummy walker :-(

Please excuse my total failure. Will test properly now...

**Comment by pkra** [comment]
I did some initial tests on Win8.1 using IE11, FF, Chrome.

This solves the initial speech string bug in NVDA and works with JAWS as well.

FWIW, the NVDA "sometimes lose first item in level" bug still persists, as does the JAWS "I start with lots of blanks until I magically stop doing that".

The earcon works with NVDA on FF and Chrome but I couldn't get it to work with JAWS on either (and the ogg won't work anywhere else). Will test more tomorrow.

**Comment by zorkow** [comment]
We can certainly change the earcon to something else. mp3 is probably a good alternative, but we may need to do some browser sniffing re FF and Opera (https://en.wikipedia.org/wiki/HTML5_Audio).

**Comment by zorkow** [comment]
This now has a selective timeout for Chrome & PC.

**Comment by pkra** [comment]
Will test this now. FYI the current PRs are diverging.

**Comment by pkra** [comment]
Preliminary summary (without WindowsXP, see above): this branch fixes NVDA's Chrome issues but no other issue. 

On the bright side, I didn't spot any regressions.

**Comment by pkra** [comment]
In case it wasn't clear, I think it's ready to merge.
